static/app/components/superuserAccessForm.tsx

```diff
@@ -59,23 +72,27 @@ class SuperuserAccessForm extends Component<Props, State> {
     const disableU2FForSUForm = ConfigStore.get('disableU2FForSUForm');
 
     const suAccessCategory = superuserAccessCategory || data.superuserAccessCategory;
-
     const suReason = superuserReason || data.superuserReason;
 
     if (!authenticators.length && !disableU2FForSUForm) {
       this.handleError(ErrorCodes.NO_AUTHENTICATOR);
       return;
     }
 
+    // Set state to setup for U2F tap
     if (this.state.showAccessForms && !disableU2FForSUForm) {
       this.setState({
         showAccessForms: false,
         superuserAccessCategory: suAccessCategory,
         superuserReason: suReason,
       });
+      // If U2F is disabled, authenticate immediately
     } else {
       try {
-        await api.requestPromise('/auth/', {method: 'PUT', data});
+        await api.requestPromise(this.props.hasStaff ? '/staff-auth/' : '/auth/', {
```

Comment:
you could pull the url into a constant outside of the function since you have the same logic in `handleU2fTap`